ui/src/pages/Access/index.tsx

```diff
@@ -58,6 +66,16 @@ export default function Access() {
     }
   };
 
+  const existingRole = useMemo(() => {
```

Comment:
Naming the variable to plural form like `existingRoles` would be better.

Reply:
Done.